src/services/ol-layer/ol-layer-factory.service.ts

```diff
@@ -47,6 +53,21 @@ export class OlLayerFactoryService {
       }
     }
 
+    this.setOlProps(<Layer>layer, olLayer)
+
+    return olLayer
+  }
+
+  createOlLayerFeature(layer: LayerFeature): OlLayer {
```

Comment:
The name might be misleading as it sounds generic, even though the decision for `position` is taken in the switch else above.

Reply:
Actually, I was thinking about using this function for drawnfeatures also (after a refactor), hence the "LayerFeature", but since I won't do this refactor soon, yes we can change the naming.
Any suggestion?

Reply:
Right, it made me also think of potential refactoring for the draw layer. So if the decision which feature layer should be created will be made within this function, the naming will make sense indeed. I was just wondering if the decision wouldn't already be made in the switch case above. In this case, we could probably just add `Position` to the name `createOlLayerFeaturePosition` ?

Reply:
renamed as suggested